_posts/en/newsletters/2020-10-07-newsletter.md

```diff
@@ -0,0 +1,198 @@
+    (script) the user wants to sign for and contains a hash commitment
+    to the desired message.  A second transaction spends the output of
+    the first transaction---if the signatures and other data for that
+    spend could be a valid transaction, then the message is considered
+    signed (although the second virtual transaction still can't be
+    included onchain because it spends from an invalid previous
+    transaction).
+
+    The advantage to using virtual transactions, which were also adopted
+    in the [BIP325][] specification for [signet][topic signet] (see
+    [Newsletter #109][news109 signet bip]), is that they may work with
+    existing software that's willing to sign arbitrary transactions,
+    including those in [PSBT][topic psbt] format.  The revised
+    specification also allows the virtual transaction to contain inputs
+    that reference specific UTXOs, allowing users to (arguably) prove
+    control over those funds, similar to [BIP127][] proof of reserves.
+
+    Alm is seeking feedback on the alternative proposal, including
+    whether or not it should replace BIP322 or be opened as a separate
+    BIP.
+
+- **Brief discussion about merging taproot in Bitcoin Core:** this
+  week's Bitcoin Core Developer Meeting included a [brief discussion][meeting
+  taproot] about whether the main taproot implementation ([PR
+  #19953][bitcoin core #19953]) should be merged in the next couple
+  weeks before the version 0.21 code is scheduled to be branched off,
+  whether it should be merged into the development branch shortly after
+  the branch off, or whether there should be no special ordering.  All
+  participants agreed any merge plans would be conditional on it
+  receiving sufficient review and no problems being found.
+
+    It seemed most participants favored merging it before the 0.21
+    branch off.  The taproot code in the PR does not contain any
+    activation parameters, so it will not be functional.  It'll be
+    similar to the way segwit code was included in [version 0.13.0][0.13
+    segwit prep] and then activation parameters were added in [version
+    0.13.1][].  To make this happen, taproot will need to finish being
+    reviewed before the project's [scheduled][bitcoin core #18947] 0.21
+    feature freeze on October 15th.  Several meeting participants said
+    they would make it their priority.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.1-beta][lnd 0.11.1-beta] is the release for a new minor
+  version.  Its release notes summarize the changes as "a number of
+  reliability improvements, some macaroon [authentication token]
+  upgrades, and a change to make our version of [anchor
+  commitments][topic anchor outputs] spec compliant."
+
+- [HWI 1.2.0-rc.1][HWI 1.2.0] is a release candidate that adds support
```

Comment:
unsure if the releases listing should be in alphabetical order or the one in the "Notable...changes" paragraph

Reply:
Primary sort order for this section is importance (as I perceive it):

- Security critical releases
- Major releases
- Minor releases
- RCs

So LND's release comes before HWI's RC.

Secondary sort order has basically been the order in which I write them, but using the notable changes order would certainly be more consistent.

Note: to keep the diffs small, I don't usually reposition RCs that get upgraded to releases after a newsletter PR is opened.